CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+
+# Question or Problem?
+
+If you have any questions or problems with the software please try posting in [Q&A](https://github.com/usnistgov/dioptra/discussions/categories/q-a).
+
+If you have general questions, please contact [dioptra@nist.gov](mailto:dioptra@nist.gov). Please allow a few business days for a response.
+
+# Submitting Issues
+
+Prior to submitting any issue please try a brief search to see if an issue similar to yours already exists. Avoiding duplicate submissions helps to limit time needed to triage issues. Each issue card should be focused and describe a single bug, feature, etc… Please try to use the [provided templates](https://github.com/usnistgov/dioptra/issues/new/choose) if they align with the issue you are submitting.
+
+## Bugs
+
+If you find a bug or unexpected behavior please use the provided [issue template](https://github.com/usnistgov/dioptra/issues/new/choose) and describe the issue along with step-by-step instructions on how to reproduce the issue.
+
+# Feature Requests
+
+Please feel free to suggest new features or capabilities. Try to describe what the desired feature will do and how it will behave and try to avoid implementation details.
+
+# Code Submission Guidelines
+
+This and following sections describe how to participate in the development process if you desire to contribute. Before starting on any large or long-term submission, please give us a heads up by sending an email to dioptra@nist.gov so that we can coordinate and avoid possible duplication of effort.
+
+Create an issue card first to spec out the change and requirements before starting work. - A pull request or update should be tied to at least one issue card (exceptions are **chore** changes like updating copyright year, files to ignore, etc.) - Keep the number of issues covered in a pull request minimal. Multiple issues being resolved in a single pull request should only happen in exceptional circumstances.
+
+Please review the sections below to better understand the code review process to insure a quick and easy acceptance of your submission.
+
+## Development Workflow
+
+Dioptra is following the [Gitflow workflow model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+[Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repository.
+
+Create a [new branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) to get started.
+
+### Squashing
+
+All final commits will be squashed, therefore when squashing your branch, it’s important to make sure you update the commit message. If you’re using Github’s UI it will by default create a new commit message which is a combination of all commits and **does not follow the commit guidelines**.
+
+If you’re working locally, it often can be useful to `--amend` a commit, or utilize `rebase -i` to reorder, squash, and reword your commits.
+
+# Commit Style Guide
```

Comment:
I think may also be a sub-heading too. Either that, or the next one should be a top-level header, although I think sub-heading may make more sense.